src/client/eppo-client.ts

```diff
@@ -497,6 +499,14 @@ export default class EppoClient implements IEppoClient {
     this.isGracefulFailureMode = gracefulFailureMode;
   }
 
+  public getConfiguration(): IUniversalFlagConfig {
+    const flags = this.configurationStore.getAll();
+    const ufc: IUniversalFlagConfig = {
+      flags,
+    };
+    return ufc;
```

Comment:
you could simplify as:
```suggestion
    return { flags };
```